pom.xml

```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.interledger</groupId>
   <artifactId>java-ilp-core</artifactId>
-  <version>0.5.0-SNAPSHOT</version>
+  <version>0.6.1-SNAPSHOT</version>
```

Comment:
im sure we've discussed this before but i seem to have forgotten - are we still checking in POMs at all? If we are using gradle, why bother?